spec/example_index.rb

```diff
@@ -15,16 +14,18 @@ class ExampleIndexWithId
   id :number
 
   mapping name: 'default', field: :name, type: :keyword, index: true
-
 end
 
-class ExampleIndexWithShard
+class ExampleIndexWithSettings
   extend ElasticSearchFramework::Index
 
-  index name: 'example_index', shards: 1
+  index name: 'example_index'
 
-  mapping name: 'default', field: :name, type: :keyword, index: true
+  payload = { custom_normalizer: { type: 'custom', char_filter: [], filter: ['lowercase'] } }
```

Comment:
What is this for?

Reply:
This is one of the options we can pass in the normalizer. I needed one for the test to validate passing multiple settings values.

We're using this so that we are normalizing the data to lowercase when we insert the strings.
Example:

The following is the order we'd want when sorting the field that holds this data.
```
Order 1
order 2
Order 3
```

Without this, the ordering is as follows:
```
order 2
Order 1
Order 3
```

Reply:
Initially I went with `value`, @adamgeorgeson had suggested `payload`. 
Do you have a preference on what we should use?